packages/google_maps_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.5.10
```

Comment:
Please rebase and update the version here and in the `pubspec.yml`.

Reply:
Done.